src/main/webapp/components/SearchBar.vue

```diff
@@ -39,8 +39,7 @@ module.exports = {
       }
       else /* Form validated */ {
         this.error = "";
-        this.queryDatabase(this.school, this.course)
-          .catch(error => this.error = error);
+        this.queryDatabase(this.school, this.course);
```

Comment:
Doesn't this conflict with commit 0708590 from PR #49?

Reply:
I think the PR has already been merged

Reply:
Oh okay, I wasn't sure if this would override that.